src/Tools/dotnet-monitor/HostBuilderHelper.cs

```diff
@@ -52,7 +52,9 @@ internal static class HostBuilderHelper
                     Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), "." + ProductFolderName) :
                     Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), ProductFolderName));
 
-        private static readonly string UserSettingsPath = Path.Combine(UserConfigDirectoryPath, SettingsFileName);
+        public static string UserSettingsPath = Path.Combine(UserConfigDirectoryPath, SettingsFileName);
+
+        public static ConfigurationTestingMode TestingMode = ConfigurationTestingMode.None;
```

Comment:
I don't believe this gets used anywhere other than CreateHostBuilder. Consider making it a parameter.